### PR TITLE
added 2.5.0-alpha-1 jre9

### PR DESCRIPTION
**Pull request by keeganwitt**

### CODE REVIEW

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 21a4e28..4c5a5c8 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -43,6 +43,7 @@ groovy:2.5-jre7
 groovy:2.5-jre7-alpine
 groovy:2.5-jre8
 groovy:2.5-jre8-alpine
+groovy:2.5-jre9
 groovy:2.5-jre-alpine
 groovy:2.5.0-alpha-1
 groovy:2.5.0-alpha-1-alpine
@@ -57,6 +58,7 @@ groovy:2.5.0-alpha-1-jre7
 groovy:2.5.0-alpha-1-jre7-alpine
 groovy:2.5.0-alpha-1-jre8
 groovy:2.5.0-alpha-1-jre8-alpine
+groovy:2.5.0-alpha-1-jre9
 groovy:2.5.0-alpha-1-jre-alpine
 groovy:alpine
 groovy:jdk
diff --git a/groovy_jre9/Dockerfile b/groovy_2.5-jre9/Dockerfile
similarity index 92%
copy from groovy_jre9/Dockerfile
copy to groovy_2.5-jre9/Dockerfile
index c859b6a..cee86d8 100644
--- a/groovy_jre9/Dockerfile
+++ b/groovy_2.5-jre9/Dockerfile
@@ -3,7 +3,7 @@ FROM openjdk:9-jre
 CMD ["groovysh"]
 
 ENV GROOVY_HOME /opt/groovy
-ENV GROOVY_VERSION 2.4.11
+ENV GROOVY_VERSION 2.5.0-alpha-1
 
 RUN set -o errexit -o nounset \
 	&& echo "Downloading Groovy" \
@@ -57,6 +57,9 @@ RUN set -o errexit -o nounset \
 	&& ln --symbolic "${GROOVY_HOME}/bin/groovysh" /usr/bin/groovysh \
 	&& ln --symbolic "${GROOVY_HOME}/bin/java2groovy" /usr/bin/java2groovy \
 	\
+	&& echo "Editing startGroovy to include java.xml.bind module" \
+	&& sed -i 's|startGroovy ( ) {|startGroovy ( ) {\n    JAVA_OPTS="$JAVA_OPTS --add-modules=java.xml.bind"|' "${GROOVY_HOME}/bin/startGroovy" \
+	\
 	&& echo "Cleaning up build dependencies" \
 	&& echo $(apt-mark showauto) \
 	&& apt-get remove --yes --purge \
````

</details>

File: library/groovy [hunk 1]
```diff
@@ -44,39 +44,44 @@ Directory: jre9
...
 
 Tags: 2.5.0-alpha-1-jdk8, 2.5-jdk8, 2.5.0-alpha-1-jdk, 2.5-jdk
 GitFetch: refs/heads/pre-release
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
 Directory: jdk8
 
 Tags: 2.5.0-alpha-1-jre8, 2.5-jre8, 2.5.0-alpha-1-jre, 2.5-jre, 2.5.0-alpha-1, 2.5
-GitCommit: 5458567bb10318d65de2675e1e374173448281e9
+GitCommit: fbf90de5f7f6f527764db38a5c6152a9139ed323
```
Comment (yosifkit) [inline library/groovy]: This one is missing its `GitFetch: refs/heads/pre-release`. It just happens to work since other tags fetch the same commit.

Comment (keeganwitt) [inline library/groovy]: Ah, good catch.  I missed that in my original commit.

**Comment by yosifkit** [comment]
Build test of #2932; 3996b3db (`groovy`):

```console

$ bashbrew build groovy:2.4.11-jdk7
Building bashbrew/cache:24db4f59dcdf1c294b915b67e7805bc1205cb36abcc70d4e2b5f65e38af2ceec (groovy:2.4.11-jdk7)
Tagging groovy:2.4.11-jdk7
Tagging groovy:2.4-jdk7
Tagging groovy:jdk7

$ test/run.sh groovy:2.4.11-jdk7
testing groovy:2.4.11-jdk7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jre7
Building bashbrew/cache:d7aa5dadb1772dc72f9753ffdae1699052ac86e928a92e45ec2adbe7e08333e3 (groovy:2.4.11-jre7)
Tagging groovy:2.4.11-jre7
Tagging groovy:2.4-jre7
Tagging groovy:jre7

$ test/run.sh groovy:2.4.11-jre7
testing groovy:2.4.11-jre7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jdk7-alpine
Using bashbrew/cache:bfac7b752476771cb3ed1736cf87dcd2c11d474793327ca51d5f4181a76a4dd6 (groovy:2.4.11-jdk7-alpine)
Tagging groovy:2.4.11-jdk7-alpine
Tagging groovy:2.4-jdk7-alpine
Tagging groovy:jdk7-alpine

$ test/run.sh groovy:2.4.11-jdk7-alpine
testing groovy:2.4.11-jdk7-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jre7-alpine
Using bashbrew/cache:42d87322ad32e370da0e3b86f633f39d7deebfc234fc8ba58b94f097f6b720dd (groovy:2.4.11-jre7-alpine)
Tagging groovy:2.4.11-jre7-alpine
Tagging groovy:2.4-jre7-alpine
Tagging groovy:jre7-alpine

$ test/run.sh groovy:2.4.11-jre7-alpine
testing groovy:2.4.11-jre7-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jdk8
Building bashbrew/cache:f02cd2a8c8c0acd69aad008f7bc6c11e4f68e0edfd734efd1f8317b8175b9b92 (groovy:2.4.11-jdk8)
Tagging groovy:2.4.11-jdk8
Tagging groovy:2.4-jdk8
Tagging groovy:jdk8
Tagging groovy:2.4.11-jdk
Tagging groovy:2.4-jdk
Tagging groovy:jdk

$ test/run.sh groovy:2.4.11-jdk8
testing groovy:2.4.11-jdk8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jre8
Building bashbrew/cache:bd99b11a4014aca2e2a0b4eb3a37feb6f5683b207056933665e0c9a443f10324 (groovy:2.4.11-jre8)
Tagging groovy:2.4.11-jre8
Tagging groovy:2.4-jre8
Tagging groovy:jre8
Tagging groovy:2.4.11-jre
Tagging groovy:2.4-jre
Tagging groovy:jre
Tagging groovy:2.4.11
Tagging groovy:2.4
Tagging groovy:latest

$ test/run.sh groovy:2.4.11-jre8
testing groovy:2.4.11-jre8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jdk8-alpine
Using bashbrew/cache:bdb4ca1c1d05e49c3cfd8e36ac39ccf9fd67da3bedea7abd378817f27fdca12a (groovy:2.4.11-jdk8-alpine)
Tagging groovy:2.4.11-jdk8-alpine
Tagging groovy:2.4-jdk8-alpine
Tagging groovy:jdk8-alpine
Tagging groovy:2.4.11-jdk-alpine
Tagging groovy:2.4-jdk-alpine
Tagging groovy:jdk-alpine

$ test/run.sh groovy:2.4.11-jdk8-alpine
testing groovy:2.4.11-jdk8-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jre8-alpine
Using bashbrew/cache:ba20b0d3e31eabdb88107298e2fa9a7129fd46804f53d8a007429d835073f03e (groovy:2.4.11-jre8-alpine)
Tagging groovy:2.4.11-jre8-alpine
Tagging groovy:2.4-jre8-alpine
Tagging groovy:jre8-alpine
Tagging groovy:2.4.11-jre-alpine
Tagging groovy:2.4-jre-alpine
Tagging groovy:jre-alpine
Tagging groovy:2.4.11-alpine
Tagging groovy:2.4-alpine
Tagging groovy:alpine

$ test/run.sh groovy:2.4.11-jre8-alpine
testing groovy:2.4.11-jre8-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jdk9
Building bashbrew/cache:e5990aabed835c61fd112d450692068c17c57257689df8df34f77d299d15bb49 (groovy:2.4.11-jdk9)
Tagging groovy:2.4.11-jdk9
Tagging groovy:2.4-jdk9
Tagging groovy:jdk9

$ test/run.sh groovy:2.4.11-jdk9
testing groovy:2.4.11-jdk9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.4.11-jre9
Building bashbrew/cache:874f7c25cfa686be490d617e9f86b35c8529d2ce1714ad12eb71cf718637b350 (groovy:2.4.11-jre9)
Tagging groovy:2.4.11-jre9
Tagging groovy:2.4-jre9
Tagging groovy:jre9

$ test/run.sh groovy:2.4.11-jre9
testing groovy:2.4.11-jre9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jdk7
Building bashbrew/cache:40948a9b1222b7eaa642ef5c5feeb095250b11e05ae33219aa5f72d1c637527c (groovy:2.5.0-alpha-1-jdk7)
Tagging groovy:2.5.0-alpha-1-jdk7
Tagging groovy:2.5-jdk7

$ test/run.sh groovy:2.5.0-alpha-1-jdk7
testing groovy:2.5.0-alpha-1-jdk7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jre7
Building bashbrew/cache:2719b6ac926654f94450691f2f33a21d0600dad86b7c972b3d16f8c825136d0a (groovy:2.5.0-alpha-1-jre7)
Tagging groovy:2.5.0-alpha-1-jre7
Tagging groovy:2.5-jre7

$ test/run.sh groovy:2.5.0-alpha-1-jre7
testing groovy:2.5.0-alpha-1-jre7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jdk7-alpine
Building bashbrew/cache:b24359b332f807ca585d4783dd7fb90d34a0285a2ff0b10e897910b8e9697034 (groovy:2.5.0-alpha-1-jdk7-alpine)
Tagging groovy:2.5.0-alpha-1-jdk7-alpine
Tagging groovy:2.5-jdk7-alpine

$ test/run.sh groovy:2.5.0-alpha-1-jdk7-alpine
testing groovy:2.5.0-alpha-1-jdk7-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jre7-alpine
Building bashbrew/cache:c9a556a1913043b4b7fa4b17bd4d01fd4c9ad2082e26b3e58d706f07cdf47e3d (groovy:2.5.0-alpha-1-jre7-alpine)
Tagging groovy:2.5.0-alpha-1-jre7-alpine
Tagging groovy:2.5-jre7-alpine

$ test/run.sh groovy:2.5.0-alpha-1-jre7-alpine
testing groovy:2.5.0-alpha-1-jre7-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jdk8
Building bashbrew/cache:ed8f07eee51042c663710c44d75bd0d0e2fc636b394f60762d461f1c5df27c75 (groovy:2.5.0-alpha-1-jdk8)
Tagging groovy:2.5.0-alpha-1-jdk8
Tagging groovy:2.5-jdk8
Tagging groovy:2.5.0-alpha-1-jdk
Tagging groovy:2.5-jdk

$ test/run.sh groovy:2.5.0-alpha-1-jdk8
testing groovy:2.5.0-alpha-1-jdk8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jre8
Building bashbrew/cache:8613ad8c0a231cb555c71b7d3215bdd7d49fa6d5fbca72f571eae2eea50173f5 (groovy:2.5.0-alpha-1-jre8)
Tagging groovy:2.5.0-alpha-1-jre8
Tagging groovy:2.5-jre8
Tagging groovy:2.5.0-alpha-1-jre
Tagging groovy:2.5-jre
Tagging groovy:2.5.0-alpha-1
Tagging groovy:2.5

$ test/run.sh groovy:2.5.0-alpha-1-jre8
testing groovy:2.5.0-alpha-1-jre8
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jdk8-alpine
Building bashbrew/cache:274c30f09b66872ae875831fad740f6bcdf26f1efa226f00bce5727fa4978443 (groovy:2.5.0-alpha-1-jdk8-alpine)
Tagging groovy:2.5.0-alpha-1-jdk8-alpine
Tagging groovy:2.5-jdk8-alpine
Tagging groovy:2.5.0-alpha-1-jdk-alpine
Tagging groovy:2.5-jdk-alpine

$ test/run.sh groovy:2.5.0-alpha-1-jdk8-alpine
testing groovy:2.5.0-alpha-1-jdk8-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jre8-alpine
Building bashbrew/cache:74278f8673f3d5b6e4a57d3cd09c85f3c180a9971cccbe805a3f746a433a4225 (groovy:2.5.0-alpha-1-jre8-alpine)
Tagging groovy:2.5.0-alpha-1-jre8-alpine
Tagging groovy:2.5-jre8-alpine
Tagging groovy:2.5.0-alpha-1-jre-alpine
Tagging groovy:2.5-jre-alpine
Tagging groovy:2.5.0-alpha-1-alpine
Tagging groovy:2.5-alpine

$ test/run.sh groovy:2.5.0-alpha-1-jre8-alpine
testing groovy:2.5.0-alpha-1-jre8-alpine
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build groovy:2.5.0-alpha-1-jre9
Building bashbrew/cache:37d16215ec9e17de1e1d39f90b391005cca3bef22544d58aad57c9234baa5d2e (groovy:2.5.0-alpha-1-jre9)
Tagging groovy:2.5.0-alpha-1-jre9
Tagging groovy:2.5-jre9

$ test/run.sh groovy:2.5.0-alpha-1-jre9
testing groovy:2.5.0-alpha-1-jre9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Comment by yosifkit** [comment]
***Note:** build server is sad today so we're waiting a little bit before merging more things to build.*

**Pull request merged**